test/with_dummyserver/test_poolmanager.py

```diff
@@ -424,6 +424,16 @@ def test_top_level_request_with_preload_content(self):
         r.data
         assert r.connection is None
 
+    def test_top_level_request_with_decode_content(self):
```

Comment:
Can we change this test to make a request with `Accept-Encoding: gzip` and `decode_content=False` and see that we get back gzipped bytes, then make another request with `decode_content=True` and see we get back decompressed bytes

Reply:
Thanks @sethmlarson 
It's done, please review when you get some time :)